### ISSUE TITLE
Older build files are not overwritten - build does not start.

### ISSUE THREAD

**Issue by stevleibelt** [stevleibelt/arch-linux-live-cd-iso-with-zfs#7]
# Given is

```
ls dynamic_data/out/
archlinux-archzfs-linux.iso  archlinux-archzfs-linux.iso.sha512sum
```
# A build is started and this is the full log

```
:: Starting cleanup build path
:: Older build detected
:: Do you want to move the files somewhere? [y|N] (n means overwriting, n is default)

:: Finished cleanup build path
:: Starting setup environment
   Previous profile data detected.
   >>/home/.../arch-linux-live-cd-iso-with-zfs/dynamic_data/releng<< exists.
   Cleaning up now ...
   Creating >>/home/.../arch-linux-live-cd-iso-with-zfs/dynamic_data<<.
   Copying content off >>/usr/share/archiso/configs/releng<< to >>/home/.../arch-linux-live-cd-iso-with-zfs/dynamic_data<
:: Finished setup environment
:: Starting evaluating environment
   Updating pacman-init.service
:: Finished evaluating environment
:: Starting adding packages and repository
:: Adding repository and package >>archzfs-linux<<.
   Using following path to the archlive >>/home/.../arch-linux-live-cd-iso-with-zfs/dynamic_data/releng<<.
:: Finished adding packages and repository
:: Starting bulding archiso
[mkarchiso] INFO: Validating options...
[mkarchiso] INFO: Done!
mv: 'archlinux-archzfs-linux.iso' and '/home/.../arch-linux-live-cd-iso-with-zfs/dynamic_data/out/archlinux-archzfs-linux

   Iso created in path:
   >>/home/.../arch-linux-live-cd-iso-with-zfs/dynamic_data/out<<
   --------
   Listing directory content, filterd by >>archzfs<<...
-rw-r--r-- 1 root root  256 Mar 24 20:03 archlinux-archzfs-linux.iso.sha512sum
-rwxrw-r-x 1 root root 867M Mar 20 18:52 archlinux-archzfs-linux.iso
   --------
:: Finished bulding archiso
:: Do you want to run the iso for testing? [y|N
```

# Current behavior

* old files are not removed
* no build was done (build was super fast)

# Expected behavior

* old build files where removed
* regular build is done

**Comment by stevleibelt** [comment]
My first thoughts are:

* implement more logging (output user input)
* add `if [[ ${?} -ne 0 ]];` for each fitting command call
* implement a `build.sh -c` flag support that acts as a `cleanup` task
* maybe implement a flag support for each step to trigger (and debug) each step on its own?

**Comment by stevleibelt** [comment]
Started adding flags with version [2.3.0](https://github.com/stevleibelt/arch-linux-live-cd-iso-with-zfs/tree/2.3.0).

```
:: Usage
   ./build.sh [-f|--force] [-h|--help] [-v|--verbose
```

**Comment by stevleibelt** [comment]
All fixed and improved with version [2.4.0](https://github.com/stevleibelt/arch-linux-live-cd-iso-with-zfs/tree/2.4.0).
